driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTransactionsTest.java

```diff
@@ -24,12 +24,23 @@
 import java.net.URISyntaxException;
 import java.util.Collection;
 
+import static com.mongodb.ClusterFixture.isSharded;
+import static com.mongodb.ClusterFixture.serverVersionLessThan;
+import static org.junit.Assume.assumeFalse;
+
 public class UnifiedTransactionsTest extends UnifiedSyncTest {
     public UnifiedTransactionsTest(@SuppressWarnings("unused") final String fileDescription,
                                    @SuppressWarnings("unused") final String testDescription,
                                    final String schemaVersion, final BsonArray runOnRequirements, final BsonArray entitiesArray,
                                    final BsonArray initialData, final BsonDocument definition) {
         super(schemaVersion, runOnRequirements, entitiesArray, initialData, definition);
+        assumeFalse(fileDescription.equals("count"));
```

Comment:
It's because all drivers used to have a `count` method in the CRUD API, but it was deprecated in favor of the pair `estimatedDocumentCount` and `countDocuments`.  Since the Java driver removed the `count` method in 4.0, we skip the still-existing tests for it that are in the specs repo for drivers that still have the method.

Reply:
Got it, thank you!